bots/discord/aoede/egg-aoede.json

```diff
@@ -0,0 +1,92 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": null
+    },
+    "exported_at": "2023-12-02T01:13:07+00:00",
+    "name": "Aoede",
+    "author": "contact@bldy.ovh",
+    "description": "Aoede is a Discord music bot that directly streams from Spotify to Discord. The only interface is Spotify itself.",
+    "features": null,
+    "docker_images": {
+        "ghcr.io\/parkervcp\/yolks:ubuntu": "ghcr.io\/parkervcp\/yolks:ubuntu"
+    },
+    "file_denylist": [],
+    "startup": ".\/aoede",
+    "config": {
+        "files": "{}",
+        "startup": "{\r\n    \"done\": \"Invite me with\"\r\n}",
+        "logs": "{}",
+        "stop": "^^C"
```

Comment:
this can be `^C` as titi is in that image ( please verify)

Reply:
I tried ^C and yes it worked as intended. I updated the file accordingly. Thanks for the feedback !